ci/docs/continuous_integation.md

```diff
@@ -10,13 +10,15 @@ HyperShift official [Documentation Guide](https://hypershift-docs.netlify.app/).
 
 ### Pre-requisites:
 
-- Install [aws](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) cli
+- Install [AWS](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) cli
+- Install [HyperShift](https://hypershift-docs.netlify.app/getting-started/#prerequisites) cli
```

Comment:
`hypershift`, as this is the cli binary, not the product.

Reply:
fixed